src/eZ/ContentView/QueryResultsInjector.php

```diff
@@ -74,7 +77,9 @@ public function injectQueryResults(FilterViewParametersEvent $event)
     private function buildResults(FilterViewParametersEvent $event): iterable
     {
         $view = $event->getView();
-        $content = $view->getContent();
+        $location = !($view instanceof LocationValueView) ?: $location = $view->getLocation();
```

Comment:
Later on you check it against `null`. Doesn't this resolve to `false` if `$view` is not an instance of `LocationValueView`?

Also, when choosing between using a negation or not use it, I think it's prefered to omit it.

```suggestion
        $location = $view instanceof LocationValueView ? $location = $view->getLocation() : null;
```

Reply:
Changed both to a regular ternary operator.